test/dynamo/test_bridge.py

```diff
@@ -0,0 +1,204 @@
+import copy
```

Comment:
I assume this test is moved from upstram `test_torchxla_integration.py`. Let's replace 
https://github.com/pytorch/xla/blob/aff3513234fe344bbf159af5e376e5a45792f5dc/test/run_tests.sh#L135

with the new path then.